Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -65,10 +70,54 @@ public partial class Autocomplete<TItem, TValue> : BaseAfterRenderComponent, ICl
         /// </summary>
         private bool closeOnSelectionAllowClose = true;
 
+        /// <summary>
+        /// True if user is using <see cref="ReadData"/> for loading the data.
+        /// </summary>
+        public bool ManualReadMode => ReadData.HasDelegate;
+
         #endregion
 
         #region Methods
 
+        protected async Task HandleReadData( CancellationToken cancellationToken = default )
+        {
+            try
+            {
+                IsLoading = true;
+
+                if ( !cancellationToken.IsCancellationRequested && IsTextSearchable )
+                    await ReadData.InvokeAsync( new( CurrentSearch, cancellationToken ) );
+            }
+            finally
+            {
+                IsLoading = false;
+
+                await InvokeAsync( StateHasChanged );
+            }
+        }
+
+        /// <summary>
+        /// Triggers the reload of the <see cref="Autocomplete{TItem, TValue}"/> data.
+        /// Makes sure not to reload if the <see cref="Autocomplete{TItem, TValue}"/> is in a loading state.
+        /// </summary>
+        /// <returns>Returns the awaitable task.</returns>
+        public async Task Reload( CancellationToken cancellationToken = default )
+        {
+            if ( IsLoading )
```

Comment:
Usually, we don't have an Is prefix.

Reply:
Based on DataGrid :) We can fix both then.

Reply:
Well I was just starting a PR to fix DataGrid... And there's a bunch of them there prefix 'Is'.  Most of them are protected, so the impact wouldn't be big, but we might have public one's....
Leave it for 2.0? Can I open an issue with this kind of cleanup?
![image](https://user-images.githubusercontent.com/22283161/172049724-edc725d9-6d46-41df-ae41-4156a8d99980.png)

I do have an habbit of using the prefix `Is`...So I guess the majority might have been introduced by me....
But even utilities we may have that pattern... And I think you're the author of these?
![image](https://user-images.githubusercontent.com/22283161/172049758-79145fa9-e804-4fed-a6ff-a80bae75c848.png)

So finish the document for Blazorise conventions first, and leave it for 2.0 (on 2.0 we try to make sure we comply to our own defined conventions)?

Reply:
Property getter are allowed to have prefix, see https://github.com/Megabit/Blazorise/issues/3424#issuecomment-1146800048

Reply:
I see.